src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const empoyees = [...document.querySelectorAll('li')];
+
+const dataAtributToNumber = (string) => {
+  const arr = string.split('');
+
+  arr.shift();
+
+  const strNormalized = arr.filter((el) => el !== ',').join('');
+
+  return +strNormalized;
+};
+
+const sortList = (list) => {
+  const parentElement = list[0].closest('ul');
+
+  list
+    .sort((value1, value2) => {
+      return (
+        dataAtributToNumber(value2.dataset.salary) -
+        dataAtributToNumber(value1.dataset.salary)
+      );
+    })
+    .forEach((employee) => parentElement.appendChild(employee));
```

Comment:
The sort function has a time complexity of O(n log n). Appending child nodes inside the sort function makes it even slower. A better approach would be to sort the array and then append all the nodes at once using DocumentFragment.